packages/core/src/components/table/table-footer/table-footer.tsx

```diff
@@ -50,6 +54,8 @@ export class TdsTableFooter {
 
   @State() horizontalScrollWidth: string = null;
 
+  @State() rowsPerPageValue: number = 10;
+
```

Comment:
What happens if a user sets a different array of values in rowsPerPageValues, for example [ 6, 12, 24, 48 ]?

Reply:
you are correct, that should be `rowsPerPageValues[0]`